tt_metal/tools/profiler/op_profiler.hpp

```diff
@@ -166,12 +166,26 @@ namespace op_profiler {
         return ret;
     }
 
+    static inline vector<json> get_tensors_json(const vector<std::optional<Tensor>>& tensors)
```

Comment:
the `const Tensor` version on line 155 could not be used?

Reply:
That was for std::optional<const Tensor>>& tensors and not std::optional<Tensor>>& tensors.  Notice the const in front of Tensors.  I can put more template magic here and reuse the same function if you think that would be better.